parts/k8s/kubernetesmastercustomscript.sh

```diff
@@ -719,3 +719,4 @@ fi
 echo `date`,`hostname`, endscript>>/opt/m
 
 mkdir -p /opt/azure/containers && touch /opt/azure/containers/provision.complete
+ps auxfww > /opt/azure/provision-ps.log
```

Comment:
Is there a reason why this is in /opt/azure and not /opt/azure/containers?

Reply:
Not really, no!